subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/strategy/StaticVersionComparator.java

```diff
@@ -24,7 +24,7 @@
 
 class StaticVersionComparator implements Comparator<Version> {
     private static final Map<String, Integer> SPECIAL_MEANINGS =
-            ImmutableMap.of("dev", new Integer(-1), "rc", new Integer(1), "final", new Integer(2));
+            ImmutableMap.of("dev", -1, "rc", 1, "final", 2, "release", 3);
```

Comment:
I think we'll want `release` < `final`. See https://groups.google.com/d/msg/gradle-dev/Z173ISH-MEQ/HkTv1P3kFgAJ.